packages/forms-web-app/src/services/submission.service.js

```diff
@@ -0,0 +1,21 @@
+const fetch = require('node-fetch');
+const logger = require('../lib/logger');
+const config = require('../config');
+
+const postSubmission = async (caseRef, body) => {
+	try {
+		const url = `${config.applications.url}/api/v1/submissions/${caseRef}`;
```

Comment:
URL may need work to get it from config ?

Reply:
All other API calls are going through the wrapper - application-api-wrapper, which handles the url and catches error responses.  Is it worth adding this there?

Reply:
There seems to be a headers argument in the handler, if it's real hassle it's no big one just be nice to have it consistent.

Reply:
the arg does not change the content type but i can change that as well. 


Will do this.